frontend/src/pages/Admin/Locations.tsx

```diff
@@ -5,16 +5,23 @@ import styles from './page.module.css';
 import Notification from '../../components/Notification/Notification';
 import LocationModal from '../../components/LocationModal/LocationModal';
 import { useLocations } from '../../context/LocationsContext';
+import SearchBar from '../../components/SearchBar/SearchBar';
+import { Tag } from '../../types';
+import { Label } from '../../components/FormElements/FormElements';
 
 const Locations = () => {
   useEffect(() => {
     document.title = 'Locations - Carriage';
   }, []);
   const [locations, setLocations] = useState<Location[]>([]);
+  const [searchedLocations, setSearchLocations] = useState(locations);
+  const [searchName, setSearchName] = useState('');
+  const [filterByTag, setFilterByTag] = useState('None');
   const loc = useLocations().locations;
 
   useEffect(() => {
     setLocations(loc);
+    setSearchLocations(loc);
   }, [loc]);
```

Comment:
Good job here! A lot of people forget to add the dependency array when using useEffect and its a huge pain lol